src/cf-forms.less

```diff
@@ -448,6 +416,86 @@ textarea {
 
 }
 
+/* topdoc
+  name: Notifications
+  family: cf-notifications
+  notes:
+    - "Markup and styling for notifications"
+  patterns:
+    - name: Default Usage
+      markup: |
+        <div class="cf-notification cf-notification__error">
+          <span class="cf-notification_icon cf-notification_icon__error cf-icon cf-icon-error-round"></span>
+          <p class="cf-notification_text">This is the default alert.</p>
+        </div>
+  tags:
+    - cfgov-misc
+*/
+
+@notification-padding__px: 11px;
+
+.cf-notification {
+    display: none;
+    position: relative;
+
+    padding: @notification-padding__px;
+    padding-left: 40px;
+    margin-bottom: @notification-padding__px + 4px;
+
+    background: @gray-20;
+    border: 1px solid @gray;
+
+    &__visible {
+      display: block;
+    }
+
+    &__error {
+        background: @redorange-20;
```

Comment:
Yep. Reposting my general comment from earlier:

All references to CFPB color variable should be converted to semantically-named color variables that can be overridden. You can probably easily copy from what Jimmy's done on his branch: https://github.com/jimmynotjim/cf-notifications/blob/initial-dev/src/less/cf-notifications.less